deltachat-ios/DC/DcMsg.swift

```diff
@@ -141,33 +116,6 @@ public class DcMsg {
         return dc_msg_get_download_state(messagePointer)
     }
 
-    public var viewtype: MessageViewType? {
-        switch dc_msg_get_viewtype(messagePointer) {
-        case 0:
-            return nil
-        case DC_MSG_AUDIO:
-            return .audio
-        case DC_MSG_FILE:
-            return .file
-        case DC_MSG_GIF:
-            return .gif
-        case DC_MSG_TEXT:
-            return .text
-        case DC_MSG_IMAGE:
-            return .image
-        case DC_MSG_STICKER:
-            return .image
```

Comment:
here was another hidden bug ... this only was okay, as at most placed, `type` was already used instead of `viewtype` :)